src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2613,18 +2613,29 @@ internal static void SetConsoleWindowTitle(string consoleTitle)
         ///
         /// </param>
         /// <exception cref="HostException">
+        /// <param name="newLine">
```

Comment:
you put the param section in an exception section.

Reply:
Fixed.